entrypoint.sh

```diff
@@ -6,9 +6,17 @@ set -e
 # If not will, the container will fail and restart.
 python3 manage.py shell -c "import django; django.db.connection.ensure_connection();"
 
-# On PROD, we run migrations at startup unless explicitly disabled.
-if [ "$ENV" == "PROD" ] && [ "${DISABLE_MIGRATION_SCRIPT_ON_PRODUCTION}" != "true" ]; then
-    scripts/migrate.sh
+# On PROD, we run migrations at startup unless explicitly disabled. If disabled,
+# this commands must be run manually
+if [ "$ENV" == "PROD" ] && [ "${DISABLE_MIGRATION_SCRIPT_ON_PRODUCTION}" != "false" ]; then
```

Comment:
Why did you change that to `"${DISABLE_MIGRATION_SCRIPT_ON_PRODUCTION}" != "false"`?

I disabled the `migrate.sh` and now you simply want to disable the `migrate` and `update_integrator_permissions`, not change the logic.